README.md

```diff
@@ -68,6 +68,23 @@ closure_repositories()
 You are not required to install the Closure Tools or PhantomJS. They will be
 fetched automatically.
 
+### Overriding Dependency Versions
+
+To override the version of any dependency, modify your `WORKSPACE` file to pass`omit_<dependency_name>=True` to `closure_repositories()`. Next define your
+custom dependency version. A full list of dependencies is available from
+[repositories.bzl](https://github.com/bazelbuild/rules_closure/tree/master/closure/repositories.bzl).
+For example, to override the version of closure-compiler:
+
+```python
+load("@io_bazel_rules_closure//closure:defs.bzl", "closure_repositories")
+closure_repositories(omit_closure_compiler=True)
+
+maven_jar(
+  name = "closure_compiler",
```

Comment:
Suggestion: Use Guava as the example. It's much more likely the user will want to omit that. I'd rather not encourage people to override Closure dependencies. They'd be better off sending us a pull request if they feel the version of Closure we're tracking is too old.